### PR TITLE
db/docs-constructor-name

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed product names in the documentation.

### CODE REVIEW

File: ts/Dashboards/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+## Working with Data
+
+You can import your data from multiple sources for example CSV and Google Spreadsheet. They are going to be handled by the Stores and distributed as DataTables.  
+
+More about this concept in the [Data documentation](https://highcharts.com/docs/dashboards/data).
+
+## GUI
+The GUI is a part of the dashboard that allows you to create a layout of the dashboard. You can add rows and cells to the layout. The layout is a grid where you can place your components.  
+An example of how the GUI configuration might look like:
+``` JS
```
Comment (goransle) [inline ts/Dashboards/README.md:53]: not really an issue, but why big JS?

**Review by goransle — APPROVED** [review]

File: ts/Dashboards/README.md (hunk 1 above)
Comment (bre1470) [reply ts/Dashboards/README.md:53]: I guess it is an abbreviation. Fun fact: You could also write JavaScript / Shell / TypeScript.

**Review by bre1470 — APPROVED** [review]
Looks good. 👍

**Comment by bre1470** [comment]
CI CY fails: Dashboards/component-add.cy.js 😕

File: ts/Dashboards/README.md (hunk 1 above)
Comment (karolkolodziej) [reply ts/Dashboards/README.md:53]: With that, the code is colored nicely ;)

**Review by pawellysy — APPROVED** [review]

**Pull request merged**